packages/components/src/utils/rtl.js

```diff
@@ -3,47 +3,90 @@
  */
 import { css } from '@emotion/core';
 
+const LOWER_LEFT_REGEXP = new RegExp( /-left/g );
+const LOWER_RIGHT_REGEXP = new RegExp( /-right/g );
+const UPPER_LEFT_REGEXP = new RegExp( /Left/g );
+const UPPER_RIGHT_REGEXP = new RegExp( /Right/g );
+
+/**
+ * Checks to see whether the document is set to rtl.
+ *
+ * @return {boolean} Whether document is RTL.
+ */
 function getRtl() {
 	return !! ( document && document.documentElement.dir === 'rtl' );
 }
 
 /**
  * Simple hook to retrieve RTL direction value
+ *
+ * @return {boolean} Whether document is RTL.
  */
 export function useRtl() {
 	return getRtl();
 }
 
+/**
+ * Flips a CSS property from left <-> right.
+ *
+ * @param {string} key The CSS property name.
+ *
+ * @return {string} The flipped CSS property name, if applicable.
+ */
+function getConvertedKey( key ) {
+	if ( key === 'left' ) {
+		return 'right';
+	}
+
+	if ( key === 'right' ) {
+		return 'left';
+	}
+
+	if ( LOWER_LEFT_REGEXP.test( key ) ) {
+		return key.replace( LOWER_LEFT_REGEXP, '-right' );
+	}
+
+	if ( LOWER_RIGHT_REGEXP.test( key ) ) {
+		return key.replace( LOWER_RIGHT_REGEXP, '-left' );
+	}
+
+	if ( UPPER_LEFT_REGEXP.test( key ) ) {
+		return key.replace( UPPER_LEFT_REGEXP, 'Right' );
+	}
+
+	if ( UPPER_RIGHT_REGEXP.test( key ) ) {
+		return key.replace( UPPER_RIGHT_REGEXP, 'Left' );
+	}
+
+	return key;
+}
+
 /**
  * An incredibly basic ltr -> rtl converter for style properties
  *
  * @param {Object} ltrStyles
+ *
  * @return {Object} Converted ltr -> rtl styles
  */
-const convertLtrToRtl = ( ltrStyles = {} ) => {
+export const convertLtrToRtl = ( ltrStyles = {} ) => {
```

Comment:
On the topic of case standards, there's also:

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#abbreviations-and-acronyms

i.e. This should technically be `convertLTRToRTL`

Reply:
Thank you!! Interesting question... In that case, should the main function be renamed from `rtl()` to `RTL()`?

Reply:
`rtl` would be the correct capitalization. The phrasing is a bit strange, but it's meant to be covered by this bit of the standard:

>If an abbreviation or an acronym occurs at the start of a variable name, it must be written to respect the camelcase naming rules covering the first letter of a variable or class definition. For variable assignment, this means writing the abbreviation entirely as lowercase.